docs/releases/upgrading/upgrade-v6.0.md

```diff
@@ -1,13 +1,157 @@
 # Upgrading from v4.0 to v6.0
 
+!!! info The title is correct
```

Comment:
Not needed (there's a comment on v5 that migration is only not available), and it will quickly be forgotten :-)